rancher/v1.5/en/kubernetes/index.md

```diff
@@ -10,9 +10,20 @@ lang: en
 
 To deploy Kubernetes in Rancher, you'll first need to create a new [environment]({{site.baseurl}}/rancher/{{page.version}}/{{page.lang}}/environments/) that has an [environment template]({{site.baseurl}}/rancher/{{page.version}}/{{page.lang}}/environments/#what-is-an-environment-template) with the container orchestration set as **Kubernetes**.
 
+### Configuring Kubernetes
+
+Kubernetes can be configured while creating or editing environment templates. If you start with a **Cattle** environment, you can configure and launch **Kubernetes** from the **Catalog** -> **Library**. If you choose the catalog route, you can skip step 1.
+
+1. Edit the Kubernetes configuration in an [environment template]({{site.baseurl}}/rancher/{{page.version}}/{{page.lang}}/environments/#what-is-an-environment-template), click on **Edit Config** in the **Orchestration** section of the template. You can edit the configuration when creating a new environment templates or when editing an existing environment template.
```

Comment:
I assume this is a new workflow in 1.5 that I am unaware of? `click on **Edit Config** in the **Orchestration** section of the template` - sounds unfamiliar to me